app/Platform/Controllers/ReportAchievementIssueController.php

```diff
@@ -41,11 +42,14 @@ public function index(Request $request, Achievement $achievement): InertiaRespon
             'game.system',
         );
 
+        $can = UserPermissionsData::fromUser($user)->include('createTriggerTicket');
+
         $props = new ReportAchievementIssuePagePropsData(
             achievement: $achievementData,
             hasSession: $foundPlayerAchievement ? true : $user->hasPlayed($achievement->game),
             ticketType: $this->determineTicketType($foundPlayerAchievement, $allPlayerAchievements),
             extra: $request->input('extra'),
+            can: $can,
         );
 
         return Inertia::render('achievement/[achievement]/report-issue', $props);
```

Comment:
```
Too few arguments to function App\Policies\TriggerTicketPolicy::create(), 1 passed in /var/www/html/vendor/laravel/framework/src/Illuminate/Auth/Access/Gate.php on line 815 and exactly 2 expected
```

Trying to click on "Report Issue" from achievement page

Reply:
Concerning that a test did not catch this.

I've fixed the issue and added a test case in latest.